pkg/apis/templating/templating.go

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Can we move this file out of the API and into `cert-manager-csi/pkg/util/template`? 

Also rename file to `template.go`.